tensorflow/core/util/tensor_format.h

```diff
@@ -121,6 +121,7 @@ inline int GetTensorSpatialDims(int num_dims, TensorFormat format) {
       // since it just a component of the width dimension.
       return num_dims - 3;  // Exclude N,C,VectDim.
   }
+  return -1;
```

Comment:
Instead of the return statement, could you please add it as a default condition to the switch with a log statement. This is the pattern followed in the rest of the file.

```
default:
      LOG(FATAL) << "Unknown format " << format;
      return -1;  // Avoid compiler warning about missing return value
```